### ISSUE TITLE
black screen after verbose

### ISSUE THREAD

**Issue by doquockhanh12** [Lovely-XPP/Dell-Latitude-E7480-Hackintosh#28]
after loading verbose, my screen goes black, but the external monitor is still on and my mouse is on it
Could you help me to fix it?
DELL Latitude 7480
CPU: i7-7600U
Intel(R) HD Graphics 620
Wifi Card: 
![image](https://user-images.githubusercontent.com/92240352/235364979-6883e044-688d-409a-8a41-b27a282d1f01.png)

**Comment by Lovely-XPP** [comment]
You can try this one first.

https://drive.google.com/file/d/1iKQ40PlBV-kVyHVCATGkO2AHL8RIURie/view?usp=share_link

**Comment by doquockhanh12** [comment]
I tried and my screen is still black

**Comment by doquockhanh12** [comment]
I tried adding -igfxblr to boot-args, my screen shows the Apple logo for a few seconds then turns to black
![z4309287710271_158848e3e9ad721355990c491d245f89](https://user-images.githubusercontent.com/92240352/235434819-2a680513-3a15-46fa-97ec-3d8054e887cf.jpg)
.

**Comment by Lovely-XPP** [comment]
Did you try to boot without the external monitor before?
BTW, you may try to change your BIOS setting to the same as the README says.

**Comment by Lovely-XPP** [comment]
> I tried adding -igfxblr to boot-args, my screen shows the Apple logo for a few seconds then turns to black
> ![z4309287710271_158848e3e9ad721355990c491d245f89](https://user-images.githubusercontent.com/92240352/235434819-2a680513-3a15-46fa-97ec-3d8054e887cf.jpg)
> .

you may also try to add `igfxonln=1` to boot-args.

**Comment by doquockhanh12** [comment]
> Did you try to boot without the external monitor before? BTW, you may try to change your BIOS setting to the same as the README says.

I tried all of them but it don't work for me

**Comment by Lovely-XPP** [comment]
I would like to know if mouse works or not when it turns black

**Comment by doquockhanh12** [comment]
> Okey, do you enter the USB installation or installed MacOS system

i enter installed MacOs system

**Comment by Lovely-XPP** [comment]
You may try to check if the system is normal or break. If you have a USB with installation, you can enter the USB installation to confirm what is the main problem, the system or OC.

**Comment by doquockhanh12** [comment]
How can i check it

**Comment by Lovely-XPP** [comment]
I think the most easy way is to enter the USB installation, if success, it is likely a problem on the system.

**Comment by doquockhanh12** [comment]
> I think the most easy way is to enter the USB installation, if success, it is likely a problem on the system.

yes, my USB installation work very well, just my screen has some error.

**Comment by Lovely-XPP** [comment]
Do you mean the screen turns black in USB installation
BTW, what config do you use for now.

**Comment by doquockhanh12** [comment]
> You can try this one first.
> 
> https://drive.google.com/file/d/1iKQ40PlBV-kVyHVCATGkO2AHL8RIURie/view?usp=share_link

i'm using it

**Comment by Lovely-XPP** [comment]
Okey, wait a minute.

**Comment by Lovely-XPP** [comment]
Try this one, overwrite the origin one:

[config.plist.zip](https://github.com/Lovely-XPP/Dell-Latitude-E7480-Hackintosh/files/11366654/config.plist.zip)

**Comment by Lovely-XPP** [comment]
or you can manually clear `Nvram` -> `7C436110-AB2A-4BBB-A880-FE41995C9F82`  -> `prev-lang:kbd`, just clear the  string, not delete the entry.

**Comment by doquockhanh12** [comment]
> Try this one, overwrite the origin one:
> 
> [config.plist.zip](https://github.com/Lovely-XPP/Dell-Latitude-E7480-Hackintosh/files/11366654/config.plist.zip)

I'm so sorry, but my screen still black

**Comment by greenfire11** [comment]
I have the same problem, did you manage to fix it

**Comment by Lovely-XPP** [comment]
I am sorry about that but I can not test the EFI on my own because I don't have not the laptop. A lot of factors will cause the problem, including USB driver, BIOS setting, hardware such as RAM and so on. Therefore it is difficult for me to give some useful suggestions for you, especially I don't have the laptop. sorry again.

**Comment by Lovely-XPP** [comment]
If you have problem to install macOS by USB, you may try recovery image.

**Comment by ZinK0** [comment]
DELL Latitude 7480
CPU: i7-7600U
Intel(R) HD Graphics 620
Intel Wifi + Bluetooth

I also have the same problem like you. After loading Apple Logo and stuck with cursor and grey screen. Nothing on monitor or LCD. Do you manage to fix that?

**Comment by AbuYousof26** [comment]
i also have the same.
did anyone tried to update the BIOS, is it may cause this issue?
